runtime/lib/backend/LightningKokkosSimulator.cpp

```diff
@@ -438,7 +438,9 @@ auto LightningKokkosSimulator::Measure(QubitIdType wire) -> Result
 
     const size_t num_qubits = this->GetNumQubits();
 
-    const auto stride = pow(2, num_qubits - (1 + wire));
+    auto &&dev_wires = this->getDeviceWires(wires);
+    const auto stride = pow(2, num_qubits - (1 + dev_wires[0]));
+    assert(std::finite(stride));
```

Comment:
No point on using this assertion here.  `getDeviceWires` always checks the validity of given wires.

```suggestion
```